### ISSUE TITLE
Setting the key to launch is tricky

### ISSUE THREAD

**Issue by stuaxo** [iamlemec/miniview#37]
For some reason I had a lot of trouble setting the hotkey, I couldn't work out where to click (clicking on the shortcut, a lot of the time just focused that area), and then when I could - setting one with a modifier, like ALT-F12, seemed impossible.

(I didn't want CTRL-F12, as that's a key in one of the emulators I use)

**Comment by iamlemec** [comment]
Yeah, the hotkey chooser has gotten janky with some GTK updates. It doesn't provide any feedback that it's working and will emit errors often. Will look at improving that, and also fixing it for GTK4.

**Comment by iamlemec** [comment]
Btw, I just pushed a newly revamped GTK4 preferences diaglog. Would be great if you got a chance to test it out! Just click on the option for for the hotkey then input the desired shortcut.
